ceph/rados/core_workflows.py

```diff
@@ -419,7 +419,10 @@ def bench_write(self, pool_name: str, **kwargs) -> bool:
         max_objs = kwargs.get("max_objs")
         verify_stats = kwargs.get("verify_stats", True)
         check_ec = kwargs.get("check_ec", True)
-        cmd = f"sudo rados --no-log-to-stderr -b {byte_size} -p {pool_name} bench {duration} write --no-cleanup"
+        cleanup = kwargs.get("cleanup", False)
+        cmd = f"sudo rados --no-log-to-stderr -b {byte_size} -p {pool_name} bench {duration} write"
+        if cleanup:
```

Comment:
Please change the variable name to `nocleanup` or change the comparison to `if not cleanup`

Reply:
Done. Updated the key to match the param name.